lib/Model/FileLock.php

```diff
@@ -196,6 +198,9 @@ public function setTimeout(int $timeout): self {
 	 * @return int
 	 */
 	public function getETA(): int {
+		if ($this->getTimeout() === 0) {
```

Comment:
Are we sure the timeout is never negative? We could say anything negative is considered as 0.